spec/support/capybara.rb

```diff
@@ -146,8 +147,9 @@ def find_sidebar
     end
 
     def expect_forbidden
-      sleep 1 # wait for the page to load
-      expect(page).to have_content("You don't have permission to access")
+      Capybara.using_wait_time 5 do
```

Comment:
nit: have you considered to use the waiting time to be in line instead of encapsulating `expect` function inside the Capybara block?

```
expect(page).to have_content("You don't have permission to access", wait: 5)
```

Reply:
Done, You're right it's more in-line with existing code.